src/lib/list/list.scss

```diff
@@ -69,7 +70,7 @@ $mat-list-item-inset-divider-offset: 72px;
   }
 
   &.mat-list-item-avatar {
-    height: $avatar-height;
+    height: $has-avatar-height;
```

Comment:
Why has this been called `$has-avatar-height`?

Reply:
Because the old name wasn't totally accurate. It wasn't "height of the avatar", but more like "height when the item has an avatar".

Reply:
Ah I see. It's just pretty confusing that the class is called `mat-list-item-avatar`. Should be in my opinion something like: `mat-list-item-with-avatar`. Also `$has-avatar-height` isn't really any more clear, just more explicit.

Reply:
Yeah I agree. I avoided renaming it in order to avoid breaking changes where people might be using a `querySelector` on it or are overriding the styling.

Reply:
Yeah, we can keep it to avoid breaking changes. If you want, having a comment on the class would still something nice IMO

Reply:
Decided to keep the old class with a deletion target and to add `mat-list-item-with-avatar`.

Reply:
👍